### PR TITLE
Add "After school program" to HoC.com dropdown

### PR DESCRIPTION
**Pull request by ebeastlake**
The following PR adds "After school program” as an option to the “Event type” dropdown on www.hourofcode.com, and (I think) ensures the string gets into the translation pipeline.

TODO:
- [x] Verify work in this PR is sufficient for translation (thanks @wilkie!)
- [x] Determine if other testing steps are needed (thanks most of Student Learning and half of Infra!)

## Links

Jira ticket: [https://codedotorg.atlassian.net/browse/SL-357](https://codedotorg.atlassian.net/browse/SL-357)

## Testing story

Tested on http://localhost.hourofcode.com:3000/ by setting `DCDO.hoc_year` to 2022 and confirming that a new submission appeared in `pegasus_development.forms` with the appropriate `event_type_s`. 

Screen Shot 2022-11-08 at 4 14 56 PM

It appears the language will fall back to English until translations are added.

Screen Shot 2022-11-08 at 3 47 04 PM

## PR Checklist:

- [x] Tests provide adequate coverage
- [x] Privacy and Security impacts have been assessed
- [x] Code is well-commented
- [x] New features are translatable or updates will not break translations
- [x] Relevant documentation has been added or updated
- [x] User impact is well-understood and desirable
- [x] Pull Request is labeled appropriately
- [x] Follow-up work items (including potential tech debt) are tracked and linked

### CODE REVIEW

**Comment by breville** [comment]
Have you confirmed that his value is successfully written to the database, and such entries show up on the map as appropriate?  More generally, just want to make sure this new value works end-to-end....

**Review by maddiedierker — COMMENTED** [review]
i'm not familiar with the Google Sheet this data is pulled from -- is there already an "after_school" column that's populated?

[edit: brendan's question is similar/better 😄]

**Comment by ebeastlake** [comment]
Outstanding work:
- [x] Verify what cron job-related work might be necessary per @sureshc's comment: https://codedotorg.slack.com/archives/C03CK49G9/p1668040346213119?thread_ts=1668040046.909849&cid=C03CK49G9
- [x] Ask why my local `pegasus_development.forms` table only seems to return one record at a time when I query using `select * from forms` (see screenshot below).
- [x] Verify that testing locally by manually setting `hoc_year` to 2022 (without actually setting from `bin/dashboard_console` was sufficient. 
- [x] Call out that I am treating "After school program" as "Organization/Company", per this discussion in Slack: https://codedotorg.slack.com/archives/C5V9YCXTR/p1668039239236899?thread_ts=1668034657.089129&cid=C5V9YCXTR

Screen Shot 2022-11-09 at 7 20 53 PM

File: pegasus/forms/hoc_signup_2022.rb [hunk 1]
```diff
@@ -4,6 +4,39 @@
 # Most of the logic hasn't changed since the 2017 form, but the 2020 form added
 # support for "at_home" as an event type.
 class HocSignup2022 < HocSignup2020
+  def self.normalize(data)
+    Honeybadger.context({data: data})
+    result = {}
+    result[:email_s] = required email_address data[:email_s]
+    result[:name_s] = required stripped data[:name_s]
+    result[:organization_name_s] = stripped data[:organization_name_s]
+    result[:event_type_s] = required enum(data[:event_type_s].to_s.strip.downcase, ['in_school', 'after_school', 'out_of_school', 'at_home'])
```
Comment (bencodeorg) [inline pegasus/forms/hoc_signup_2022.rb:13]: nit: could you add a comment here describing how this is different than 2020 (I think its just this line)?

File: apps/src/sites/hourofcode.com/pages/public/index.js [hunk 2]
```diff
@@ -125,7 +125,9 @@ $(document).ready(function() {
       $('#hoc-entire-school').show();
       $('#continue-btn').hide();
       $('#submit-btn').show();
-    } else if ($('#hoc-event-type').val() === 'out_of_school') {
+    } else if (
+      ['out_of_school', 'after_school'].includes($('#hoc-event-type').val())
+    ) {
       // out of school, either US or non-US
```
Comment (bencodeorg) [inline apps/src/sites/hourofcode.com/pages/public/index.js]: nit: update (or remove, seems self-explanatory) comment here?

**Review by bencodeorg — APPROVED** [review]
A couple of comment nits but LGTM!

I think this is fine to merge without verifying the map (sounds like you and Bethany looked at the logic that populates the map and there's no indication that event type would impact what gets shown, and seems complicated to setup a test). Maybe worth throwing a reminder on the calendar though to check out some events on the map next week to confirm they're there?

File: pegasus/forms/hoc_signup_2022.rb (hunk 1 above)
Comment (ebeastlake) [reply pegasus/forms/hoc_signup_2022.rb:13]: Yes, meant to do that, thanks Ben!

File: apps/src/sites/hourofcode.com/pages/public/index.js (hunk 2 above)
Comment (ebeastlake) [reply apps/src/sites/hourofcode.com/pages/public/index.js]: We could probably remove all of these comments, but I updated this one instead to note that after school and out of school events are handled the same way under the hood.

**Review by wilkie — APPROVED** [review]
Looks good from the i18n side! Maybe I'll even get this in the sync tonight.

I defer to others on the content.

**Pull request merged**